[ERROR] BlhostBase: Unsupported protocol version: P1.3.0

how to update ?

```
pyblhost serial ping -p /dev/ttyUSB0 -b 115200
[INFO] BlhostBase: Sending ping command
[INFO] BlhostBase: Ping response: version: P1.3.0, options: 0
[ERROR] BlhostBase: Unsupported protocol version: P1.3.0
[INFO] Ping responded in 1 attempt(s)
```

--- a/pyblhost/pyblhost.py
+++ b/pyblhost/pyblhost.py
@@ -539,7 +539,7 @@ class BlhostBase(object):
             protocol_bugfix, protocol_minor, protocol_major, protocol_name, options = struct.Struct('<BBBBH').unpack(data[2:8])
             protocol_version = '{}{}.{}.{}'.format(chr(protocol_name), protocol_major, protocol_minor, protocol_bugfix)
             self.logger.info('BlhostBase: Ping response: version: {}, options: {}'.format(protocol_version, options))
-            if protocol_version != 'P1.2.0':
+            if protocol_version not in ['P1.2.0', 'P1.3.0']:
                 self.logger.error('BlhostBase: Unsupported protocol version: {}'.format(protocol_version))
         else:
             self.logger.info('BlhostBase: Unhandled command type: {}'.format(data[1]))

